pkg/lib/page.scss

```diff
@@ -213,4 +213,11 @@ body {
   .pf-v6-c-backdrop {
     --pf-t--global--background--color--backdrop--default: var(--pf-t--global--background--color--secondary--default) !important;
   }
+
+  // We set svg aliasing to precision instead of auto/optimizeSpeed to try to ensure
+  // consistent rendering between test runs.
+  svg {
```

Comment:
Should we make this .pixel-tests svg? So it only applies to our pixel tests?

Reply:
Yes, absolutely, otherwise the "real" svgs would look crap. But it is already in that section.